mautic/mautic
# Fix special characters in form condition value

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:

Noticed If parent select box use special characters in value, compare function does not work properly for conditions. 


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create from with selectbox and value `čé+äà>&"`
3. Then make conditional field with condition for value in `čé+äà>&"`
4. Check the form and submit
5. See like your children field is not save to result properly because it's skipped
6. After PR should work properly




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11093"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
